examples/conflicts/server.js

```diff
@@ -1,10 +1,18 @@
 const express = require('express')
 const queries = require('./queries')
 const { VoyagerServer, gql } = require('@aerogear/voyager-server')
+const metrics = require('@aerogear/voyager-metrics')
 
 const { conflictHandler } = require('@aerogear/voyager-conflicts')
 
-conflictHandler.enableLogging(console)
+conflictHandler.setConflictListener({
```

Comment:
Really like it. We can improve that even better once #81  will be added.

Reply:
@wtrocki which one gets merged first?